Makefile

```
@@ -0,0 +1,295 @@
+		$(_SRC_MOUNT_RW) \
+		$(DEBUG_IMAGE) \
+		bunx biome format --write src/
+
+# =============================================================================
+# Image management targets
+# =============================================================================
+
+## rm_containers: Stop and remove every container created from whap images.
+# Queries both the production image and the debug image so containers from
+# either build are cleaned up.  Safe to run when no containers exist.
+rm_containers:
```

checkmake (0.3.2)

[warning] Target body for "rm_containers" exceeds allowed length of 5 lines (7).

(maxbodylength)